Typescript — TS2308: Module "./default" has already exported a member named 'Ret'

Typescript 4.9.3 reports:
```
node_modules/typegram/index.d.ts:10:1 - error TS2308: Module "./default" has already exported a member named 'Ret'. Consider explicitly re-exporting to resolve the ambiguity.

10 export * from "./proxied";
   ~~~~~~~~~~~~~~~~~~~~~~~~~~
Errors  Files
     1  node_modules/typegram/index.d.ts:10
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -7,5 +7,5 @@ export * from "./menu-button";
 export * from "./message";
 export * from "./passport";
 export * from "./payment";
-export { Typegram } from "./proxied";
+export { type Typegram } from "./proxied";
 export * from "./update";

--- a/index.d.ts
+++ b/index.d.ts
@@ -7,5 +7,5 @@ export * from "./menu-button";
 export * from "./message";
 export * from "./passport";
 export * from "./payment";
-export * from "./proxied";
+export { Typegram } from "./proxied";
 export * from "./update";

